app/models/vm_or_template.rb

```diff
@@ -336,6 +337,15 @@ def run_command_via_parent(verb, options = {})
     ext_management_system.send(verb, self, options)
   end
 
+  def run_command_via_task(task_options, queue_options)
+    MiqTask.generic_action_with_callback(task_options, command_queue_options(queue_options))
+  end
```

Comment:
Do we want to return only the task ID or the full task object here? I would generally prefer the latter, unless the UI needs the former for some reason.

https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_task.rb#L303

Reply:
This maintains compatibility with what is currently being returned, https://github.com/ManageIQ/manageiq/pull/19544/files#diff-6002b94a939143e9dcfa4873f9fe003bL87

I'm okay changing it but we need to find and update all the callers as well.